pages/s/_id.vue

```diff
@@ -362,7 +362,7 @@ export default class ScholarProfile extends Vue {
 
     .age
       margin-left: 10px
-      color: lighten($sch-purple, 40%)
+      color: $sch-purple3
```

Comment:
In darkmode lighten will be either almost imperceptible or basically turn white. `$sch-purple3` is a better alternative - imo.
See also, TeamCard.vue